### DISCUSSION TITLE
Can't run tests AND script (import problems)

### DISCUSSION THREAD

**Discussion by botkero** [pytest-dev/pytest#10627] [Q&A] [answered]
Hello,

I have the problem that my imports doesn't work when I try both testing and running.

My project structure is like this:
```md
src/
  - __init__.py
  - main.py
  - driver/
     -  __init__.py
     - many stuff
tests/
  - __init__.py
  - test_main.py

```

To start my tests, I use the following command:  
`pytest -v --junitxml=report.xml --cov-config=.coveragerc --cov=src/ --cov-report term-missing`
```py
# test_main.py
from src.driver import Driver

# main.py
from src.driver import Driver
```

When I start my main.py script normally, then I need to change this
`python3 src/main.py`
```py
# main.py
from driver import Driver
```
Otherwise, I get the error:
> ModuleNotFoundError: No module named 'src'

All my `__init__.py` files are empty except of the one in driver/.
Do you have any ideas how to resolve my problem?

**Comment by The-Compiler** [comment, marked as answer]
Run your code as a Python module instead: `python -m src.main`

**Reply by botkero** [reply]
Thanks!!!
